src/components/__tests__/Superscript.ts

```diff
@@ -46,3 +46,17 @@ it.skip('Superscript should not count archived contexts', async () => {
   const element = screen.getByText('2')
   expect(element.nodeName).toBe('SUP')
 })
+
+it('Superscript should not count for hashed version of metaprogramming attributes like =archive | archive', async () => {
+  store.dispatch([
+    importText({
+      text: `
+      - a
+        - =archive
+      - b
+        - Archive`,
+    }),
+  ])
+
+  expect(() => screen.getByText('2')).toThrow('Unable to find an element')
```

Comment:
You should always make sure your test fails on the old code first.

This test will always pass because `=archive` is not visible, so it will not contribute to the `Superscript` count. You have to `toggleHiddenThoughts` to make it visible for checking the `Superscript`.